src/notifications.js

```diff
@@ -77,7 +77,14 @@ export default class Notification {
 
   mapCombineTopics (topics) {
     const prefixes = {};
-    topics.map(t => {
+    const precombinedTopics = [];
+    topics.filter(t => {
+      if (t.indexOf('?') === -1) {
+        return true;
+      } else {
+        precombinedTopics.push(t);
```

Comment:
Also, this is a side-effect and should generally be avoided. By that, I mean you shouldn't do 
```
something.filter((item) => {
  if (!item.condition) {
    // do something unrelated
  }

  return item.condition
}
```
I understand *why* you are doing this and that it saves a line or two of code, but I'd prefer either using a forEach to create 2 different arrays, or running a separate dedicated filter to find the precombinedTopics. Does that make sense what I'm saying?

Reply:
Yep, it does, and will do!